dockerfiles/attestation-service/Dockerfile

```diff
@@ -10,7 +10,9 @@ COPY patches/ patches/
 COPY packages/base/package.json packages/base/
 COPY packages/utils/package.json packages/utils/
 COPY packages/typescript/package.json packages/typescript/
+COPY packages/typechain-target-web3-v1-celo/package.json packages/typechain-target-web-v1-celo/
```

Comment:
Actually, we have a fork that I created a time ago (https://github.com/celo-org/typechain-target-web3-v1-celo) because we had typings problems with the the web3 types extensions. And as we have it as an actual package, docker was just installing it, that's why it was not needed to copying it anymore. 
Now that typechain uses our communication library for the typing, I basically added to the repo for two things:
a) it was easier to generate changes and test the flows without re publishing our typechain a lot of times
b) as it is a custom made for us that is only used by the protocol package, I thought that having it in the monorepo could be easier to add changes.

But, as I made all the class generation part is stable, I could add the changes in our fork, publish it again, and use it as a dependency
(I'll change that)

Reply:
Removed